elasticio

```diff
@@ -31,4 +32,11 @@ program
     await runExec(path, func, fixture, action);
   });
 
+program
+  .command('cmp:validate', 'Validate component.json, action/trigger files, and any static schemas')
+  .argument('<path>', 'Path to component folder')
```

Comment:
We should actually do this for all the commands! It will take a bit of reworking of other stuff david currently is editing though, so i'll leave it for a different pr :)